Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -855,7 +855,12 @@ public override void RunAction(Act act)
                 if (act.GetType() == typeof(ActBrowserElement))
                 {
                     ActBrowserElement ABE = (ActBrowserElement)act;
-                    if (ABE.ControlAction == ActBrowserElement.eControlAction.SwitchToDefaultWindow)
+                    if (ABE.ControlAction == ActBrowserElement.eControlAction.AcceptMessageBox)
```

Comment:
try to just add IF that will check the action type + operation and if it one of those mentioned here so skip the rest of this method and jump to the DoRunAction.
if it works so please show to Jinendra to see if we missed any scenario.